### PR TITLE
Add summary page

### PR DESCRIPTION
**Pull request by k-sullivan**
Introducing the summary page, featuring:
- Updated donut (SummaryCard) component*
- Unrolled donut list (DmarcReportSummaryGraph) component*
- Six tables (DmarcReportTable) with pagination and ordering*
- Translations
*Component data will be retrieved using demo queries

image
image
image

**Now with bonus date translations and sorting for DmarReportSummaryGraph
image

### CODE REVIEW

File: frontend/src/DmarcReportSummaryGraph.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  const { periods, strengths } = data
+  const ticks = [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
+  const { colors } = theme
+
+  const formatTicks = (tick) => {
+    return `${tick * 100}%`
+  }
+
+  // Create date entry for x-axis data keys (e.g.: 'JAN-19')
+  // TODO: Should we just be receiving the data in this format?
```
Comment (nslandolt) [inline frontend/src/DmarcReportSummaryGraph.js:51]: At the time I was not sure how you wanted this so I assumed you wanted the full month name, I can make the change if you would like.

**Comment by k-sullivan** [comment]
> Looks good, just wondering with the screen shot of the churro chart, the dates are out of order, is that something from my end, or is that from faker? and you can just ping me in slack regarding the other comment.

Added forced sorting in 9db2434

File: frontend/src/DmarcReportTable.js [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+  ${(props) =>
+    props.flatHeaders.slice(1).map((headerObj, index) => {
+      return `
+        td:nth-of-type(${index + 1}):before {
+          content: '${headerObj.Header}';
+          font-weight: bold;
+        }
+      `
+    })}
+`
```
Comment (sleepycat) [inline frontend/src/DmarcReportTable.js:105]: This component isn't getting it's styles from our [theme](https://github.com/canada-ca/tracker/blob/d2688b6f7841648c88cd3f40e31c64d6cd0ddbd0/frontend/src/index.js#L10). Chakra UI is [built around the idea of a theme](https://chakra-ui.com/getting-started), [an idea that it inheirits from Styled-System](https://styled-system.com/getting-started#theming) by way of building on top of it. That theme supplies all the breakpoints widths, colours and other stuff, which this component is defining for itself, creating a little "style island". In service of "gathering feedback" this is fine, but needs a follow up PR to switch it over to drawing it's values from the theme or this is going to get out of hand quickly.

File: frontend/src/App.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { SkipLink } from './SkipLink'
 import { TwoFactorNotificationBar } from './TwoFactorNotificationBar'
 import { useUserState } from './UserState'
 import { RouteIf } from './RouteIf'
+import { DmarcReportPage } from './DmarcReportPage'
```
[the remark this answers is not in the document]
Comment (k-sullivan) [reply frontend/src/App.js]: Fixed in de89749

**Pull request merged**